lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -151,17 +163,14 @@ def callback(ctxt, param, value):
         "--name",
         expose_value=False,
         callback=callback,
-        help=(
-            "list the tool groups in which <tool-name> is used.\n"
-            "Not allowed with the --group option"
-        ),
```

Comment:
Nice! we do need to update the help message in pbench-clear-tool (as this messages are from the list tools command) :+1:

Reply:
Thanks.  (I'm a little fuzzy on the second part...is this all good?)